server/svix-server/src/worker.rs

```diff
@@ -76,7 +76,7 @@ impl FailureCacheKey {
 ///
 /// If the key value pair does not already exist in the cache, indicating that the endpoint never
 /// stopped responding, no operation is performed.
-async fn process_success_cache(
+async fn process_endpoint_success(
     cache: &Cache,
     app_id: &ApplicationId,
     endp_id: &EndpointId,
```

Comment:
Since you're refactoring, I'd probably also pass `org_id` here and fix the cache key for failure to include the org_id. I know app_id is globally unique, but having good segmentation between different orgs is probably wanted.

Reply:
Should we have a migration here to accommodate prior keys?